packages/legacy/core/App/utils/helpers.ts

```diff
@@ -267,20 +258,21 @@ export function getConnectionName(
   )
 }
 
-export function getCredentialConnectionLabel(credential?: CredentialExchangeRecord) {
+export function useCredentialConnectionLabel(credential?: CredentialExchangeRecord) {
+  const connection = useConnectionById(credential?.connectionId ?? '')
+
   if (!credential) {
     return ''
   }
 
   if (credential.connectionId) {
-    const connection = useConnectionById(credential.connectionId)
     return connection?.alias || connection?.theirLabel || credential.connectionId
   }
 
   return 'Unknown Contact'
 }
 
-export function getConnectionImageUrl(connectionId: string) {
+export function useConnectionImageUrl(connectionId: string) {
```

Comment:
Checking, this is a hook ya? AFAIK only hooks should start with `use`?

Reply:
It is a hook, as it uses a hook, so it had to be named with `use`